<?xml version="1.0"?>

<!-- map.launch -->
<launch>

    <node name="differential_drive" pkg="igvc_navigation" type="differential_drive" output="screen" required="true">
        <param name="axle_length" value="0.376"/> <!-- .188 * 2 -->
        <param name="max_vel" value="2.0"/>
    </node>

</launch>
